src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -38,6 +40,9 @@ public class WebScriptHostManager : ScriptHostManager
         private readonly ScriptHostConfiguration _config;
         private readonly ISwaggerDocumentManager _swaggerDocumentManager;
         private readonly object _syncLock = new object();
+
+        private readonly WebJobsSdkExtensionHookProvider _bindingWebhookProvider = new WebJobsSdkExtensionHookProvider();
```

Comment:
can the type here and below be the base IWebHookProvider type? I'd expect that callers should only need members defined by the interface, not subclass specific members.